src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedFieldFactory.js

```diff
@@ -44,13 +44,37 @@ export const FixedFieldFactory = {
     }
 
     config.fields = fixedFieldType.items.filter(x => !x.readOnly).map((item) => {
+      // Temporary fix API for type mapa and field Proj
+      if (item.code === 'Proj') {
+        item.isArray = false;
+      }
```

Comment:
The BE part is already merged, I think this can be removed.

Reply:
Yes - it was merged but not installed on snapshot it still has version "marc-records-editor 5.3" but should have 5.4.
As soon as it will be installed I'll do the changes also related with 006 specyfication for Proj fileld.